pandas/tseries/resample.py

```diff
@@ -395,8 +397,8 @@ def _groupby_and_aggregate(self, how, grouper=None, *args, **kwargs):
 
         return self._wrap_result(result)
 
-    def _apply_loffset(self, result):
-        """if loffset if set, offset the result index"""
+    def _apply_loffset(self, result, arg=None):
```

Comment:
this is very confusing, why do we need ``arg``? pls add a ``Parameters`` section